tmp/canary.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: canary-service
+  labels:
+    app: canary
+    strimzi.io/kind: Canary
+spec:
+  ports:
+  - port: 8080
+    protocol: TCP
+  selector:
+    app: canary
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: canary-deployment
+spec:
+  selector:
+    matchLabels:
+      app: canary
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: canary
+        strimzi.io/kind: Canary
+    spec:
+      containers:
+      - name: canary
+        image: strimzi-canary:0.0.1
+        ports:
+        - containerPort: 8080
+          name: rest-api
```

Comment:
I think we can only add this part to `packaging/install` because deployment and service is already defined there

Reply:
yes. When I opened this PR some months ago, the installation folder was not there, so I created this temporary deployment descriptor in this PR. I'll update it to the current state.